src/Tribe/Rewrite.php

```diff
@@ -2,24 +2,18 @@
 // Don't load directly
 defined( 'WPINC' ) or die;
 
+/**
+ * Rewrite Configuration Class
+ * Permalinks magic Happens over here!
+ */
+class Tribe__Events__Rewrite extends  Tribe__Rewrite {
 	/**
-	 * Rewrite Configuration Class
-	 * Permalinks magic Happens over here!
-	 */
-	class Tribe__Events__Rewrite {
-		/**
-		 * If we wish to setup a rewrite rule that uses percent symbols, we'll need
-		 * to make use of this placeholder.
-		 */
-		const PERCENT_PLACEHOLDER = '~~TRIBE~PC~~';
-
-		/**
-		 * Static singleton variable
+	 * Static singleton variable
 	 * @var self
-		 */
-		public static $instance;
+	 */
+	public static $instance;
 
-		/**
+	/**
```

Comment:
This line is tabbed correctly but much of what follows is still indented by one extra tab.